api/database/migrations/2021_09_17_190139_create_pool_candidate_filters_table.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class CreatePoolCandidateFiltersTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('pool_candidate_filters', function (Blueprint $table) {
+            $table->uuid('id')->primary('id');
+            $table->timestamps();
+            $table->boolean('is_woman')->nullable(true);
+            $table->boolean('has_diploma')->nullable(true);
+            $table->boolean('has_disability')->nullable(true);
+            $table->boolean('is_indigenous')->nullable(true);
+            $table->boolean('is_visible_minority')->nullable(true);
+            $table->string('language_ability')->nullable(true);
+            $table->jsonb('work_region')->nullable(true);
+            $table->softDeletes();
+        });
+        DB::statement('ALTER TABLE pool_candidate_filters ALTER COLUMN id SET DEFAULT gen_random_uuid();');
+        Schema::create('operational_requirement_pool_candidate_filter', function (Blueprint $table) {
+            $table->uuid('id')->primary('id');
+            $table->timestamps();
+            $table->string('pool_candidate_filter_id', 36)->references('id')->on('pool_candidate_filters')->nullable(false);
+            $table->string('operational_requirement_id', 36)->references('id')->on('operational_requirements')->nullable(false);
+        });
+        DB::statement('ALTER TABLE operational_requirement_pool_candidate_filter ALTER COLUMN id SET DEFAULT gen_random_uuid();');
+        Schema::create('classification_pool_candidate_filter', function (Blueprint $table) {
+            $table->uuid('id')->primary('id');
+            $table->timestamps();
+            $table->string('pool_candidate_filter_id', 36)->references('id')->on('pool_candidate_filters')->nullable(false);
+            $table->string('classification_id', 36)->references('id')->on('classifications')->nullable(false);
```

Comment:
feature/add-candidate-search-requests-model
```suggestion
            $table->uuid('pool_candidate_filter_id')->references('id')->on('pool_candidate_filters')->nullable(false);
            $table->uuid('classification_id')->references('id')->on('classifications')->nullable(false);
```